### PR TITLE
Tickets #1468 and #1685

### LINKED ISSUE #1685: Fix Other menu item info INFO
The menu "Other" doesn't work properly in INFO.  This is because there is no Node: section for it.  There seems to be some strange INFO formatting in this file -- not sure why -- where topics are duplicated.  I won't fix that, but I will add a Node: for Other so that it can be found from the top-level menu.

### PR DESCRIPTION
**Pull request by eswenson1**
Resolves: #1468. Adds INFO topic Tips that includes various tips on using and administering ITS, including one on how to debug OJB devices.

Resolves #1685: "Other" menu in INFO directory doesn't work properly.

### CODE REVIEW

File: doc/info/dir.230 [hunk 1]
```diff
@@ -22,7 +22,7 @@ File: DIR	Node: Top	This is the top of the INFO tree
 * Debugging facilities::
 * Terminal Support::
 * ITS: ITS
-* Simulators::
+* Simulators::
```
Comment (larsbrinkhoff) [inline doc/info/dir.230]: Is this added space needed?

File: doc/info/dir.231 [hunk 2]
```diff
@@ -24,6 +24,7 @@ File: DIR	Node: Top	This is the top of the INFO tree
 * ITS: ITS
 * Simulators:: 
 * Other::
+* Tips: (ejs;tips >)Top
```
Comment (larsbrinkhoff) [inline doc/info/dir.231]: The TIPS file is missing from the commit, right?

**Comment by eswenson1** [comment]
Whoops, I did this so many times I forgot tips and the fact that I moved it to info:. Will fix

**Comment by larsbrinkhoff** [comment]
I moved the space character fix from the second to the first commit, rebased, and force pushed.

**Pull request merged**